study.md

```diff
@@ -40,7 +40,7 @@ In your own words, define what the responsibilities of the controller layer are
 in Rails.
 
 ```md
-<!-- your answer here -->
+A controller takes the user's input and figures out how to process it and what, if anything, to send back. This is where the methods live.
```

Comment:
It's where *some* methods live. The actions defined inside controllers determine what is done before sending a response.